scripts/gef.sh

```diff
@@ -22,13 +21,19 @@ if [ -f "${HOME}/.gdbinit" ]; then
 fi
 
 if [ $wget_found -eq 1 ]; then
+    latest_tag=$(wget -q -O- "https://api.github.com/repos/hugsy/gef/tags" | grep "name" | head -1 | sed -e 's/"name": "\([^"]*\)",/\1/' -e 's/\s*//')
```

Comment:
I imagine you need that same fix each time you build `latest_tag` e.g. here and line 33.

Reply:
damn osx